### DISCUSSION TITLE
Was not able to create login credentials after installation

### DISCUSSION THREAD

**Discussion by felixKroni** [telekom-security/tpotce#1582] [Q&A]
Hello,
I´ve installed t-pot on a cloud vm (vultr) using an iso version 22.04.0.
I went through the whole installation process twice now and i can set language and keyboard settings and so on but im not able to set a username or password. Once the installation of the iso is complete i remove the iso and restart the VM. Using the "View Console" in vultr i can access the console but instead of the installation of t-pot i get a login view.
![image](https://github.com/telekom-security/tpotce/assets/32167792/f21c536d-c07c-4c3a-ae55-f781f890deb0)

I have no clue what pw i can use. is there some kind of default?
ive looked at two tutorials now both using vultr and there it works perfectly.
One of the Tutorials: https://medium.com/@SatyamPathania/t-pot-honeypot-setup-ultimate-vps-guide-and-documentation-for-cybersecurity-students-0432637bb5e2

Any Ideas what ive done wrong?

**Comment by t3chn0m4g3** [comment]
You have to use 24.04, all other versions are no longer supported.

**Comment by felixKroni** [comment]
is there anything different in the installation?
